src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/global.yml

```diff
@@ -317,3 +317,41 @@ Resources:
               - "*"
       Roles:
         - !Ref ReadOnlyAutomationRole
+  ADFTerraformRole:
```

Comment:
This role and the next policy should be removed from this file.
As they are moved to a commented out version in the `example-global-iam.yml` file, for the user to enable when they want to use Terraform.

Reply:
removed